Unix/configure

```diff
@@ -154,6 +155,9 @@ do
       certsdir=$arg 
       ;;
 
+    --credsdir=*)
+      credsdir=$arg 
+      ;;
```

Comment:
I think this will go away if read from the server. But if it stays, all other stuff seems to have a blank line after `;;`, you should after this section too.